src/utils/validations.js

```diff
@@ -194,3 +194,38 @@ export const validateBmcURL = value => {
 
   return validateURL(value);
 };
+
+// Validates that the provided MAC address meets one of following formats supported by the golang ParseMAC function:
+// IEEE 802 MAC-48, EUI-48, EUI-64, or a 20-octet IP over InfiniBand link-layer address
+// https://golang.org/pkg/net/#ParseMAC
+export const isValidMAC = mac => {
+  if (mac.length < 14) {
+    return false;
+  }
+
+  let isValid = false;
+  if (mac[2] === ':') {
+    const regex = new RegExp(
```

Comment:
Can we extract a `const HEXCH_REGEX = '[0-9A-Fa-f]'` variable out of these? 

Would be even better to have a function which creates a regex e.g. `f(groupLength, numOfGroups, delimiter)`

Reply:
I created regexes for each of the delimiter types for which we're looking. That made things look much cleaner.

If we created a function like the one you suggested, we'd have to run the function three times for each of the delimiter regexes and then concatenate those results to capture all three possible lengths. I think that would be more confusing than just having the regexes as they are with the comments showing the format that each part of the regex is trying to match.

I also thought about the hex character regex, but I think it would decrease readability to mix the regex with string interpolation like that.

Reply:
ok, but IMO for example just `HEXCH` is more readable than `[0-9A-Fa-f]` especially when it is used multiple times in the same regex. And is less prone to mistakes.

String interpolation is quite readable by itself so I don't see any problem mixing it with regex

Reply:
I disagree, but you seem to have a stronger opinion about it than I do, so I made the change. =)

Reply:
I guess the readability is again subjective, but I hope you agree with the reusability/less mistakes (smaller area) part.